assets/js/wpmenucart-wc-block-support.js

```diff
@@ -0,0 +1,48 @@
+jQuery( function( $ ) {
+    let wpmenucart_ajax_timer;
+    function WPMenucart_Load_AJAX() {
+        let data = {
+            security:   wpmenucart_cart_ajax.nonce,
+            action:     "wpmenucart_ajax",
+        };
+
+        xhr = jQuery.ajax({
+            type:       'POST',
+            url:        wpmenucart_cart_ajax.ajaxurl,
+            data:       data,
+            success:    function( response ) {
+                jQuery('.wpmenucartli').html( response );
+            }
+        });
+    }
+    wp.hooks.addAction(
+        `experimental__woocommerce_blocks-cart-set-item-quantity`,
+        'wpmenucart-cart',
+        function() {
+            clearTimeout( wpmenucart_ajax_timer );
+            wpmenucart_ajax_timer = setTimeout( WPMenucart_Load_AJAX, 3000);
+        }
+    );
+
+    wp.hooks.addAction(
+        `experimental__woocommerce_blocks-cart-remove-item`,
```

Comment:
It would be beneficial to add the function to the "experimental__woocommerce_blocks-cart-add-item" hook to address a situation where a page contains both products and a cart. In this scenario, if a user adds a product to their cart, the menu cart will not update.